RankingTests/RankingTests.cs

```diff
@@ -33,30 +36,67 @@ public void TestGetScore()
             bool actual = rank.Scores.Contains(score2);
             Assert.AreEqual(true, actual, "Test ranking can add score");
         }
+        //test: check if file can be loaded from
+        //expected: file is missing so cannot load (false)
         [Test]
-        public void TestLoadFileFalse()
+        public void TestCheckFileMissing()
         {
             Ranking rank = new Ranking();
-            bool actual = rank.LoadFile("scorenope.txt");
-            Assert.AreEqual(false, actual, "Test ranking can load score from file");
+            bool actual = rank.CheckFile("scorenope.txt");
+
+            Assert.AreEqual(false, actual, "Test ranking can detect missing file");
+        }
+        //test: check if file can be loaded from
+        //expected: file has more than 11 lines so cannot load (false)
+        [Test]
+        public void TestCheckFileValid1()
```

Comment:
? add more text files is just for testing purpose
and i dont think we should read the file to the list of score if its not checked that it contains only digits
also why bother reading it to a list of file when you can just check if it's more than 10
the way i see it: 
1. game load: load ranking from file (if invalid format/more than 10, empty ranking)
2. game win/lose: add score to ranking, sort, cut to 10
3: game close: overwrite file

Reply:
Keep in mind that we start with the empty method. Therefore, if we read multiple text files for testing purpose, it seems to be unit testing. If you're thinking about what if we have no file at the beginning of the game, you can implement the SaveToFile method first.

My point of view:
1. I think we can check the result after reading the file. If the result is false, the file will be invalid. Otherwise, the file will be valid.
2. Or you can check line by line (using the while loop to read a line and check whether the result is non-digit or digit). In this case, we don't need to save the result into the list of scores.

If this is still not clear, we can discuss today.

Reply:
Okay ill keep it to 1 text file.
Yes i plan it to check it line by line in the checkfile method